### DISCUSSION TITLE
Bull MQ vs Trigger Dev?

### DISCUSSION THREAD

**Discussion by deadcoder0904** [triggerdotdev/trigger.dev#922] [Q&A]
My simple application right now only needs 1 function (to send email after payment is done) & I think Bull MQ would be fine. What do you think? I have never used Queues before so don't know much about it.

Also, would love to know what are the benefits of using Trigger Dev over just simply using Bull MQ?

In what use-cases would Trigger Dev be preferred over Bull MQ?

Is Bull MQ Pro similar to Trigger Dev?

**Comment by rafalzawadzki** [comment]
if you only need to send an email then you don't need neither trigger nor bull :)

**Reply by deadcoder0904** [reply]
@rafalzawadzki i recently [watched some system design](https://www.youtube.com/watch?v=sSKhdZ32YpY) concepts (first time for me in 10 years) but it said that you should put your non-critical tasks in a queue.

so i thought i should put my **sending email** task in a queue after getting payment. is that a correct idea?

note: i dropped bullmq/trigger both from my project about 2-3 days ago as i thought it would be overkill for my small project.

i'm curious if most system design concepts are used/required in only big apps with lots of requests/scale/concurrency?

**Reply by rafalzawadzki** [reply]
you could use a queue if you had thousands of emails to send - for example an email notification on a social network, where one action notifies many users at once. Don’t overcomplicate your small app :)

**Reply by deadcoder0904** [reply]
got it, thanks. i won't mark it as an answer as i'd still like the original question to be answered.
